### PR TITLE
feat(payment): PI-2403 Pay by Bank integration

### PR DESCRIPTION
**Pull request by bc-ania**
## What?
Pay By Bank is a supported payment method - [Pay by Bank (Open Banking)](https://developers.bluesnap.com/reference/pay-by-bank) 

Allow shoppers to input IBAN in the Pay By Bank payment step

## Why?
To implement Pay by Bank method

## Testing / Proof
Screenshot 2024-08-26 at 10 29 14

Screenshot 2024-08-26 at 12 10 28

...

### CODE REVIEW

File: packages/bluesnap-direct-integration/src/BlueSnapDirectPayByBankPaymentMethod.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+    return (
+        <Fieldset
+            legend={
+                <Legend hidden>{language.translate('payment.bluesnap_direct_iban.label')}</Legend>
+            }
+            style={{ paddingBottom: '1rem' }}
+        >
+            <BlueSnapDirectTextField
```
Comment (vitalii-koshovyi) [inline packages/bluesnap-direct-integration/src/BlueSnapDirectPayByBankPaymentMethod.tsx:28]: should it be just one input? do we need some permission checkbox or something?

Comment (bc-ania) [reply packages/bluesnap-direct-integration/src/BlueSnapDirectPayByBankPaymentMethod.tsx:55]: no permission box needed, just one input field for IBAN

**Review by bc-nick — APPROVED** [review]

**Review by vitalii-koshovyi — CHANGES_REQUESTED** [review]
please add e2e test

**Comment by bc-ania** [comment]
> please add e2e test

@vitalii-koshovyi test added

**Review by bc-peng — APPROVED** [review]

**Review by vitalii-koshovyi — APPROVED** [review]

**Pull request merged**